app/api/helpers/files.py

```diff
@@ -113,6 +114,8 @@ def create_save_resized_image(image_file, basewidth=None, maintain_aspect=None,
     uploaded_url = upload(upfile, upload_path)
     os.remove(temp_file_path)
 
+    uploaded_url = re.sub(r'\:\d+/', '/', uploaded_url)
```

Comment:
Is this the right place to strip the port? I'm unsure as this is outside the test framework. Also, this is different from the code for stripping in the case of protocol-port matches you mentioned earlier (`http`: `80` and so on). I'll add code for it after this part gets reviewed :)

Reply:
I don't think you should really strip the port. This is the uploaded url being returned, it should have the port as well. You should check where the file is being loaded, the path should not have port in it